spa/templates/basic-auth.js

```diff
@@ -1,26 +1,29 @@
 // Expected Authorization header value
 const authorization = "Basic ${credentials}"
 
+// List of regular expressions describing paths excluded from the basic auth
+const exclusions = "${exclusions}".split(" ").filter(function (e) {return e})
```

Comment:
you could avoid this somewhat brittle code by passing in the array of exclusions as a JSON string, so that in terraform

```tf
exclusions = jsonencode(var.basic_auth_exclusions)
```

and in the lambda code

```suggestion
const exclusions = ${exclusions}
```

https://www.terraform.io/docs/language/functions/jsonencode.html

Reply:
I like it 👍🏻